src/autocomplete.js

```diff
@@ -31,30 +31,30 @@ export default class Autocomplete {
 
     this.mouseDown = false
 
-    this.onInputChange = debounce(this.onInputChange.bind(this), 300)
-    this.onResultsMouseDown = this.onResultsMouseDown.bind(this)
-    this.onInputBlur = this.onInputBlur.bind(this)
-    this.onInputFocus = this.onInputFocus.bind(this)
-    this.onKeydown = this.onKeydown.bind(this)
-    this.onCommit = this.onCommit.bind(this)
+    this.onInputChangeHandler = debounce(this.onInputChange.bind(this), 300)
```

Comment:
I don't mind this change but if you wanna keep the diff smaller you can pull in `babel-preset-github@2.1.2` which fixes the `.bind(this)` error.